Java/Assinment_5/First.java

```diff
@@ -0,0 +1,22 @@
+package com.company.data;
+
```

Comment:
Below is a general feedback to be followed across the repo: 
1. Code readability and comments to be worked on for implementation of SOLID as we had discussed earlier like grouping functionalities and all.
2. Git and Folder Structure : 
        - Please give proper description(not so long) while making any commits next time onwards.
        - Also if you have any Jira story, can include id in the commit message(optional)
        - Please dont make any changes like file modification/delete or any other from git directly, always try to push and raise PR's from local.
        - Maintain proper folder structure for your Java Code/Any Assignments given, give proper folder names. Would be good for everyone when anyone visits them.
        	- ask anyone how they are doing or maybe the LDP team.
        
3. Please go through modules/lecture that are not covered yet, for clean code, sonar as discussed. Ignore if you have completed.
4. Can see StringBuilder content in the git repo for assignment, make sure you are comfortable with all that is written over there. Would be appreciated if written in a language of your understanding. 
        Also as you already read about it, make sure to just have hands on the functions and its working.
        - as discussed pick any problem statement related to localdatetime eg difference between two dates, day for any given date or pick any problem from your side
           and make sure you understand them properly, and commit the code with a problem statement in the comments

Reply:
Noted !